h5py/tests/common.py

```diff
@@ -94,9 +95,9 @@ def assertSameElements(self, a, b):
             if not match:
                 raise AssertionError("Item '%s' appears in b but not a" % x)
 
-    def assertArrayEqual(self, dset, arr, message=None, precision=None):
+    def assertArrayEqual(self, dset, arr, message=None, precision=None, ignore_alignment=False):
```

Comment:
Nitpick: could we flip the sense of the parameter, so the default is something like `check_alignment=True`, and you pass False to skip it. I prefer True to mean the positive sense.